cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -69,12 +69,21 @@ def self.config(env)
             proxy: 'dashboard',
             headers: LANGUAGE_HEADER,
             cookies: whitelisted_cookies
+          },
+          {
+            path: %w(
+              /
+              /learn*
+              /congrats
+            ),
+            headers: LANGUAGE_HEADER,
+            cookies: LANGUAGE_COOKIE
           }
         ],
-        # Default Pegasus paths are cached but language-specific, whitelist only language cookie/header.
+        # Remaining Pegasus paths are English-only and don't require any extra headers or cookies.
```

Comment:
Is english-only the right default assumption?  When someone adds a new pegasus path you can assume by default that they will forget to update this file. Or even if they do, that they'll miss some other step in deploying the change.  It's bad when users see random languages.